crates/sui-swarm-config/tests/snapshots/snapshot_tests__network_config_snapshot_matches.snap

```diff
@@ -44,6 +43,34 @@ validator_configs:
           send_certificate_rate_limit: ~
           report_batch_rate_limit: ~
           request_batches_rate_limit: ~
+      parameters:
+        db_path: ~
+        leader_timeout:
+          secs: 0
+          nanos: 250000000
+        min_round_delay:
+          secs: 0
+          nanos: 50000000
+        max_forward_time_drift:
+          secs: 0
+          nanos: 500000000
```

Comment:
I am not so sure if that's a convenient format to use on the property files. In this case I would encourage using/copy over the parser we currently have in NW https://github.com/MystenLabs/sui/blob/main/narwhal/config/src/duration_format.rs so we deal with more human friendly expressions (ex 200ms etc..)

Reply:
The formatting definitely improves, but I'm not sure if implementing ser/de for Duration can have other unintended consequences.